qemu/tests/block_copy.py

```diff
@@ -314,12 +314,15 @@ def wait_for_finished(self):
         """
         waiting until block job finished
         """
+        time_start = time.time()
         params = self.parser_test_args()
         timeout = params.get("wait_timeout")
         finished = utils_misc.wait_for(self.job_finished, timeout=timeout)
         if not finished:
             raise error.TestFail("Job not finished in %s seconds" % timeout)
+        time_end = time.time()
```

Comment:
@xiywang, Thanks. Since I need to count time several times when calling  wait_for_finished, it will cause duplication code if handle it outside. And other part use this function does not need to be modified, just ignore the return value is ok. So I prefer add it here.